alert-database/src/test/java/com/synopsys/integration/alert/database/api/mock/DefaultMockJPARepository.java

```diff
@@ -0,0 +1,140 @@
+package com.synopsys.integration.alert.database.api.mock;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public class DefaultMockJPARepository<T, ID> implements JpaRepository<T,ID> {
+    // Empty implementation of a JPA repository to be extended by other mock repositories.
+    // The child classes that extend this can then override the methods they need to
+    // provide the correct testing functionality needed.
```

Comment:
These lines may not be necessary if you made the class abstract. Having the keyword there would others realize that the class should be extended.